tests/src/dir.common.props

```diff
@@ -27,13 +27,13 @@
 
 <!-- Setup the default output and intermediate paths -->
   <PropertyGroup>
-    <BaseOutputPathWithConfig>$(ProjectDir)\..\binaries\tests\$(Platform)\$(Configuration)\</BaseOutputPathWithConfig>
+    <BaseOutputPathWithConfig>$(ProjectDir)\..\binaries\tests\$(Platform).$(OS).$(Configuration)\</BaseOutputPathWithConfig>
```

Comment:
While this following the directory convention it is not consistent with the order. The order in corefx is $(OS).$(Platform).$(Configuration)\$(MSBuildProjectName).

That is true in all the places through this review. I do think there is good reason to be consistent with the ordering on all these as well.